install-requirements.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Better to call this file just `install-requirements` (no suffix), since it is executable with a shebang.
https://google.github.io/styleguide/shellguide.html#file-extensions

Reply:
done